content/wiki/preparing_office_hours.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "Preparing Office Hours"
++++
+
+If you've volunteered to host Office Hours, the first step will be to create the agenda document.
+
+Create a copy of [this template file](https://docs.google.com/document/d/10wUKjCAEIvYCQLnp_QpdFrAXkO0sBnarG2abbI7mduQ/edit?tab=t.0) in the [Office Hours Google drive folder](https://drive.google.com/drive/folders/1mmrRnlpYb3j5P0ewAOcY_zj2tTH5u5aO) (you may need additional permissions).
+
+You then need to fill out the sections about the different projects, listing the important changes.
+
+You may want to use a script like the following to programmatically get a list of recently-changed issues and PRs:
```

Comment:
I'm not convinced that `script` here really sufficiently implies that you should use `deno`.

Reply:
Reworded the intro text.